awx/ui/templates/ui/index.html

```diff
@@ -9,14 +9,14 @@
     <title ng-bind="tabTitle"></title>
     <script>var $basePath = '{{ STATIC_URL }}'</script>
     
-        <link href="{{ STATIC_URL }}css/vendor.f283b4795bc6921b48e5.css" rel="stylesheet" />
+        <link href="{{ STATIC_URL }}css/vendor.60b270d4310f8d01aab3.css" rel="stylesheet" />
```

Comment:
Did you intend for these things to change?  I've seen this happen before too @gconsidine any idea what's up here?

Reply:
Whoops no I did not. I can revert that.

Reply:
I think this is automatically generated by some of our tooling and I definitely want to know why (I just ran into it also).

Reply:
@matburt I think this started happening after #49 landed.

Reply:
`index.html` is generated now. The hash in the filename changes when the UI is built for cache busting. I'm adding `index.html` to `.gitignore` in a separate PR.